unittests/forked_tests_savanna.cpp

```diff
@@ -0,0 +1,717 @@
+   BOOST_REQUIRE_EQUAL( trace1->id, _nodes[0].get_unapplied_transaction_queue().begin()->id() );
+   BOOST_REQUIRE_EQUAL( trace2->id, (++_nodes[0].get_unapplied_transaction_queue().begin())->id() );
+   BOOST_REQUIRE_EQUAL( trace3->id, (++(++_nodes[0].get_unapplied_transaction_queue().begin()))->id() );
+   BOOST_REQUIRE_EQUAL( trace4->id, (++(++(++_nodes[0].get_unapplied_transaction_queue().begin())))->id() );
+
+   BOOST_REQUIRE_EXCEPTION(_nodes[0].control->get_account( "test1"_n ), fc::exception,
+                           [a="test1"_n] (const fc::exception& e)->bool {
+                              return std::string( e.what() ).find( a.to_string() ) != std::string::npos;
+                           }) ;
+   BOOST_REQUIRE_EXCEPTION(_nodes[0].control->get_account( "test2"_n ), fc::exception,
+                           [a="test2"_n] (const fc::exception& e)->bool {
+                              return std::string( e.what() ).find( a.to_string() ) != std::string::npos;
+                           }) ;
+   BOOST_REQUIRE_EXCEPTION(_nodes[0].control->get_account( "test3"_n ), fc::exception,
+                           [a="test3"_n] (const fc::exception& e)->bool {
+                              return std::string( e.what() ).find( a.to_string() ) != std::string::npos;
+                           }) ;
+   BOOST_REQUIRE_EXCEPTION(_nodes[0].control->get_account( "test4"_n ), fc::exception,
+                           [a="test4"_n] (const fc::exception& e)->bool {
+                              return std::string( e.what() ).find( a.to_string() ) != std::string::npos;
+                           }) ;
+
+   // produce block which will apply the unapplied transactions
+   produce_block_result_t produce_block_result = _nodes[0].produce_block_ex(fc::milliseconds(config::block_interval_ms), true);
+   std::vector<transaction_trace_ptr>& traces = produce_block_result.unapplied_transaction_traces;
+
+   BOOST_REQUIRE_EQUAL( 4u, traces.size() );
+   BOOST_CHECK_EQUAL( trace1->id, traces.at(0)->id );
+   BOOST_CHECK_EQUAL( transaction_receipt_header::executed, traces.at(0)->receipt->status );
+   BOOST_CHECK_EQUAL( trace2->id, traces.at(1)->id );
+   BOOST_CHECK_EQUAL( transaction_receipt_header::executed, traces.at(1)->receipt->status );
+   BOOST_CHECK_EQUAL( trace3->id, traces.at(2)->id );
+   BOOST_CHECK_EQUAL( transaction_receipt_header::executed, traces.at(2)->receipt->status );
+   // test4 failed because it was tapos to a forked out block
+   BOOST_CHECK_EQUAL( trace4->id, traces.at(3)->id );
+   BOOST_CHECK( !traces.at(3)->receipt );
+   BOOST_CHECK( traces.at(3)->except );
+
+   // verify unapplied transactions ran
+   BOOST_REQUIRE_EQUAL( _nodes[0].control->get_account( "test1"_n ).name,  "test1"_n );
+   BOOST_REQUIRE_EQUAL( _nodes[0].control->get_account( "test2"_n ).name,  "test2"_n );
+   BOOST_REQUIRE_EQUAL( _nodes[0].control->get_account( "test3"_n ).name,  "test3"_n );
+
+   // failed because of tapos to forked out block
```

Comment:
`tapos`?

Reply:
That was in the original `forked_tests.cpp` test, comment was just copied over.

Reply:
https://eosio.stackexchange.com/questions/2362/what-is-transaction-as-proof-of-stake-tapos-and-when-would-a-smart-contract

Reply:
Do you know who is `confused00` (who answered this question)?